workers/unity/Assets/Playground/Scripts/Player/ProcessScoresSystem.cs

```diff
@@ -0,0 +1,38 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Collections;
+using Unity.Entities;
+
+namespace Playground
+{
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
+    class ProcessScoresSystem : ComponentSystem
+    {
+        private struct ScoringData
+        {
+            public readonly int Length;
+            public ComponentDataArray<SpatialOSLauncher> Launcher;
+
+            [ReadOnly]
+            public ComponentArray<CommandRequests<Generated.Playground.Launcher.ScoreIncrease.Request>> CommandRequests;
```

Comment:
do we need the whole namespace here

Reply:
Unfortunately, we do :(

Reply:
Can put a using at the top if it affects readability.